cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -128,11 +128,11 @@
         .optimize-loading { opacity: 0 !important; }
     </style>
     <script>
-        (function(a,s,y,n,c,h,i,d,e){s.className+=' '+y;h.start=1*new Date;
-        h.end=i=function(){s.className=s.className.replace(RegExp(' ?'+y),'')};
-        (a[n]=a[n]||[]).hide=h;setTimeout(function(){i();h.end=null},c);h.timeout=c;
-        })(window,document.documentElement,'optimize-loading','dataLayer',5000,
-        {'GTM-KHB8MB':true});
+      (function(a,s,y,n,c,h,i,d,e){s.className+=' '+y;h.start=1*new Date;
```

Comment:
Is this indentation change intentional? HTML files should still be four space, even if writing JS

Reply:
It was intentional. Once you're inside `<script>` it's JS land, no?

Reply:
I don't think so. If we linted it, the linter would only care that the _file_ was html, not what the code written inside of it was.

Reply:
Okay I reverted it, but I think a smart HTML linter wouldn't be touching what's inside a `<script>` tag because it's not HTML. If we used something like https://www.npmjs.com/package/eslint-plugin-html#htmlindent, we could control it however we want, but it would still be JS and JS code style I'd think.